rir/src/compiler/translations/pir_2_rir/pir_2_rir.cpp

```diff
@@ -1420,9 +1426,8 @@ rir::Function* Pir2Rir::finalize() {
 
     assert(signature.formalNargs() == cls->nargs());
     ctx.push(R_NilValue);
-    size_t localsCnt = compileCode(ctx, cls);
+    auto body = compileCode(ctx, cls);
```

Comment:
ok, this is slightly unrelated, because I pulled those changes out of my native branch.

The api was a bit strange in the first place. Why should a compile method return as the result the number of locals the code needs? that is very accidental api design.

I had to change it for native compilation and thought, why not fix it already now..

Reply:
👍